guides/plugins/plugins/administration/add-mixins.md

```diff
@@ -0,0 +1,69 @@
+# Adding Mixins
+
+This documentation chapter will cover how to add a new administration Mixin for your plugin.
+Mixins behave generally the same as they do normally in Vue, just the registration and how mixins are included in a component is different.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files and a running plugin. Of course you'll have to understand JavaScript, but that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation. As stated before mixins in Shopware are basically the same as in Vue so you should have read their [documentation](https://vuejs.org/v2/guide/mixins.html) on them first. 
+
+## Register a new service
+
+For this example, we'll just use the example mixin from the [VueJS documentation](https://vuejs.org/v2/guide/mixins.html) and adjust it to be used in Shopware.
+
+Mixins in Shopware have to be registered in the mixin registry via the `Mixin.register` function to available everywhere in the administration.
+
+Converting the Vue mixin to be used in Shopware and looks like the example seen below:
+
+{% code title="<administration root>/mixins/swag-basic-example.js" %}
```

Comment:
Please emphasize the changes you did, e.g. via comment (if there aren't that many)

Reply:
Well, there are no real changes, they used an intermediary variable and i'm directly feeding the mixin into the register function

Reply:
Ok, some comments won't hurt then. 👍